Leibit.Core/Data/sued/TF.xml

```diff
@@ -0,0 +1,412 @@
+    <track name="1">
+      <block name="65G1"/>
+      <alternative track="2"/>
+    </track>
+    <track name="2">
+      <block name="65G2"/>
+      <alternative track="1"/>
+    </track>
+  </station>
+
+  <station name="Ludwigshafen (Bodensee)" short="RLU" refNr="68" scheduleFile="g68_____.abf" localOrderFile="bf68____.abf">
+    <track name="1">
+      <block name="68G1"/>
+      <alternative track="2"/>
+    </track>
+    <track name="2">
+      <block name="68G2"/>
+      <alternative track="1"/>
+    </track>
+  </station>
+
+  <station name="Sipplingen" short="RSIN" refNr="68" scheduleFile="sippli__.abf">
+    <track name="N">
+      <block name="68BN"/>
+    </track>
+  </station>
+
+  <station name="Überlingen" short="RUEB" refNr="69" scheduleFile="g69_____.abf" localOrderFile="bf69____.abf">
+    <track name="1">
+      <block name="69G1"/>
+      <alternative track="2"/>
+      <alternative track="3"/>
+    </track>
+    <track name="2">
+      <block name="69G2"/>
+      <alternative track="1"/>
+      <alternative track="3"/>
+    </track>
+    <track name="3">
+      <block name="69G3"/>
+      <alternative track="2"/>
+      <alternative track="1"/>
+    </track>
+  </station>
+
+  <!--ScheduleFile prüfen, RUEO hat 2!-->
+  <station name="Überlingen-Ost" short="RUEO" refNr="69" scheduleFile="überli__.abf">
+    <track name="N">
+      <block name="69BN"/>
+    </track>
+  </station>
+
+  <station name="Uhldingen-Mühlhofen" short="RUDM" refNr="70" scheduleFile="g70_____.abf" localOrderFile="bf70____.abf">
+    <track name="1">
+      <block name="70G1"/>
+      <alternative track="2"/>
+    </track>
+    <track name="2">
+      <block name="70G2"/>
+      <alternative track="1"/>
+    </track>
+  </station>
+
+  <station name="Salem" short="RSLM" refNr="71" scheduleFile="g71_____.abf" localOrderFile="bf71____.abf">
+    <track name="1">
+      <block name="71G1"/>
+      <alternative track="2"/>
+      <alternative track="3"/>
+    </track>
+    <track name="2">
+      <block name="71G2"/>
+      <alternative track="1"/>
+      <alternative track="3"/>
+    </track>
+    <track name="3">
+      <block name="71G3"/>
+      <alternative track="2"/>
+      <alternative track="1"/>
+    </track>
+  </station>
+
+  <station name="Bermatingen-Ahausen" short="RBAH" refNr="71" scheduleFile="bermat__.abf">
+    <track name="N">
+      <block name="71BN"/>
+    </track>
+  </station>
+
+  <station name="Markdorf" short="RMAD" refNr="72" scheduleFile="g72_____.abf" localOrderFile="bf72____.abf">
+    <track name="1">
+      <block name="72G1"/>
+      <alternative track="2"/>
+    </track>
+    <track name="2">
+      <block name="72G2"/>
+      <alternative track="1"/>
+    </track>
+  </station>
+
+  <station name="Friedrichshafen Stadt" short="TF" refNr="77" scheduleFile="g77_____.abf" localOrderFile="bf77____.abf">
+    <track name="1">
+      <block name="77G1"/>
+      <alternative track="2"/>
```

Comment:
Hier sind die Alternativ-Gleise genau richtig und auch notwendig, da der Bahnhof in Pbf und Gbf geteilt ist. 👍